Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ActionsLibraryNavPage.xaml.cs

```diff
@@ -259,7 +259,7 @@ private void grdActions_PreviewDragItem(object sender, EventArgs e)
             if (DragDrop2.DragInfo.DataIsAssignableToType(typeof(Act)))
```

Comment:
Need to remove grdActions_PreviewDragItem from here because we do not drop on it